opsdroid/connector/github/__init__.py

```diff
@@ -32,16 +32,20 @@ def __init__(self, config, opsdroid=None):
 
     async def connect(self):
         """Connect to GitHub."""
-        url = "{}/user?access_token={}".format(GITHUB_API_URL, self.github_token)
-        async with aiohttp.ClientSession(trust_env=True) as session:
-            response = await session.get(url)
-            if response.status >= 300:
-                _LOGGER.error(_("Error connecting to GitHub: %s."), response.text())
+        url = "{}".format(GITHUB_API_URL)
+        headers = {"Authorization": "token {}".format(self.github_token)}
```

Comment:
f-Strings might be preferred here.

Reply:
Yeah we would prefer f-strings moving forwards.